boxcli/shell.go

```diff
@@ -47,3 +48,11 @@ func runShellCmd(cmd *cobra.Command, args []string) error {
 	}
 	return err
 }
+
+func nixShellPersistentPreRunE(cmd *cobra.Command, args []string) error {
```

Comment:
If you want to simplify the name, something like `shellPreRun` would be fine too.

Reply:
This was actually unrelated to this PR. I had to merge again to fix the diff.